requirements-integration.txt

```diff
@@ -2,14 +2,14 @@
 # This file is autogenerated by pip-compile with Python 3.8
 # by the following command:
 #
-#    pip-compile requirements-integration.in
+#    pip-compile --resolver=backtracking ./requirements-integration.in
```

Comment:
Yes, I know that, but using this resolver has meant issues in the past and we try to avoid it, that's why I would like to know if it gave you trouble when running w/o specifying the resolver.

Reply:
I just run tox to upgrade the deps

and what is the issue with new resolver?
I assume there should be more issues with legacy